[2019-04-04 01:55:42,530] {/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py:1761} ERROR - Exception on /booking/12 [PUT]
Traceback (most recent call last):
  File "/home/alphatech/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 1978, in delete
    state = attributes.instance_state(instance)
AttributeError: 'NoneType' object has no attribute '_sa_instance_state'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 104, in wrapper
    return fn(*args, **kwargs)
  File "/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/booking/resources.py", line 171, in put
    db.session.delete(qry_jadwal)
  File "/home/alphatech/.local/lib/python3.6/site-packages/sqlalchemy/orm/scoping.py", line 162, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 1980, in delete
    raise exc.UnmappedInstanceError(instance)
sqlalchemy.orm.exc.UnmappedInstanceError: Class 'builtins.NoneType' is not mapped
[2019-04-04 01:55:42,534] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:42} WARNING - REQUEST_LOG	{"request": {"status": "cancelled"}, "response": {"message": "Internal Server Error"}}
[2019-04-04 01:56:54,578] {/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py:1761} ERROR - Exception on /booking/12 [PUT]
Traceback (most recent call last):
  File "/home/alphatech/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 1978, in delete
    state = attributes.instance_state(instance)
AttributeError: 'NoneType' object has no attribute '_sa_instance_state'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 104, in wrapper
    return fn(*args, **kwargs)
  File "/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/booking/resources.py", line 171, in put
    db.session.delete(qry_jadwal)
  File "/home/alphatech/.local/lib/python3.6/site-packages/sqlalchemy/orm/scoping.py", line 162, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 1980, in delete
    raise exc.UnmappedInstanceError(instance)
sqlalchemy.orm.exc.UnmappedInstanceError: Class 'builtins.NoneType' is not mapped
[2019-04-04 01:56:54,585] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:42} WARNING - REQUEST_LOG	{"request": {"status": "cancelled"}, "response": {"message": "Internal Server Error"}}
[2019-04-04 01:57:10,010] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:42} WARNING - REQUEST_LOG	{"request": {"status": "cancelled"}, "response": "tes"}
[2019-04-04 01:58:47,759] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:42} WARNING - REQUEST_LOG	{"request": {"status": "cancelled"}, "response": 12}
[2019-04-04 02:01:20,242] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:42} WARNING - REQUEST_LOG	{"request": {"username": "Aswindanu", "password": "Warcr4ft"}, "response": {"status": "success", "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE1NTQzMTgwODAsIm5iZiI6MTU1NDMxODA4MCwianRpIjoiNDg5OTY0OTktMWZiZC00ZDhhLWFiOWQtNzk0MzU2ZjkwOWU1IiwiZXhwIjoxNTU0NDA0NDgwLCJpZGVudGl0eSI6eyJpZCI6MTEsInRpcGUiOiJjbGllbnQifSwiZnJlc2giOmZhbHNlLCJ0eXBlIjoiYWNjZXNzIiwidXNlcl9jbGFpbXMiOnsiaWQiOjExLCJ0aXBlIjoiY2xpZW50In19.1w1gGjL4k9_v12m8dNFHkeDtFKxAa1hUZc6CJ7XGsuQ", "stat": "client"}}
